js/plugin-loader.js

```diff
@@ -438,15 +445,15 @@ ProxyConnectorPlugIn.prototype = {
     if (!(super.isValid(context) && this.host && this.port)) {
       return false;
     }
-    //we should not load authentication types that are 
-    //not requested by the administrator
-    if (!context.authManager.authPluginRequested(this.identifier,
-      this.authenticationCategory)) {
-      bootstrapLogger.warn("Authentication plugin was found which was not requested in "
-          + "the server configuration file's dataserviceAuthentication object. "
-          + "Skipping load of this plugin");
-      return false;
-    }
+//    //we should not load authentication types that are 
+//    //not requested by the administrator
+//    if (!context.authManager.authPluginRequested(this.identifier,
+//      this.authenticationCategory)) {
+//      bootstrapLogger.warn("Authentication plugin was found which was not requested in "
```

Comment:
Why was this commented out? we were actually using this.

Reply:
It looked misplaced: why does this have to be under the "proxy connector"?

Reply:
Nevermind, you are right. Its a duplicate block and I was thinking of the duplicate: https://github.com/zowe/zlux-proxy-server/blob/e66c15d44ae5632dfb7fd8a37849e3aea41856eb/js/plugin-loader.js#L404